modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -192,7 +198,106 @@ $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_worker-machi
 ----
 +
 Because you create and manage the worker machines yourself, you do not need to initialize these machines.
+//The current AWS UPI method removes all of the `99_openshift-cluster-api_worker-machineset-* files. What is different about the Local Zones that means that we should retain them? Should we remove whatever the installation program creates and then specifically create this new file?
 endif::aws,azure,ash,gcp[]
+ifdef::localzone[]
+. Create the machine set manifests for the worker nodes in your Local Zone.
+.. Export a variable to define the instance type for the worker machine to deploy on the Local Zone subnet by running the following command:
++
+[source,terminal]
+----
+$ export INSTANCE_TYPE="<instance_type>" <1>
+----
+<1> Set `<instance_type>` to a tested instance type, such as `c5d.2xlarge`.
+
+.. Store the AMI ID as a local variable by running the following command:
++
+[source,terminal]
+----
+$ export AMI_ID=$(grep ami \
+  openshift/99_openshift-cluster-api_worker-machineset-0.yaml \
+  | tail -n1 | awk '{print$2}')
+----
+
+.. Store the AMI ID as a local variable by running the following command:
++
+[source,terminal]
+----
+$ export SUBNET_ID=$(aws cloudformation describe-stacks \
+  --stack-name "<subnet_stack_name>" \ <1>
+  | jq -r .Stacks[0].Outputs[0].OutputValue)
+----
+<1> For `<subnet_stack_name>`, specify the name of the subnet stack that you created.
+
+.. Create the worker manifest file for the Local Zone that your VPC uses by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF > openshift/99_openshift-cluster-api_worker-machineset-nyc1.yaml
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: ${CLUSTER_ID}
+  name: ${CLUSTER_ID}-edge-${LZ_ZONE_NAME}
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: ${CLUSTER_ID}
+      machine.openshift.io/cluster-api-machineset: ${CLUSTER_ID}-edge-${LZ_ZONE_NAME}
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: ${CLUSTER_ID}
+        machine.openshift.io/cluster-api-machine-role: edge
+        machine.openshift.io/cluster-api-machine-type: edge
+        machine.openshift.io/cluster-api-machineset: ${CLUSTER_ID}-edge-${LZ_ZONE_NAME}
+    spec:
+      metadata:
+        labels:
+          location: local-zone
+          zone_group: ${LZ_ZONE_NAME::-1}
```

Comment:
Let's keep it in sync with the standardized [Enhancement](https://github.com/openshift/enhancements/pull/1232/files#diff-f2590494c9138551ab5b3d5632d11170fe8742a128256f9e1d8ebc63baa19f41R531-R532) and [implementation](https://github.com/openshift/installer/pull/6371/files#diff-75b6aa907d64f8fa958f3d4baffa4200d309771e666a121547ede93bca0e1b3dR57-R58):

```suggestion
          zone_type: local-zone
          zone_group: ${LZ_ZONE_NAME::-1}
```